client/src/reducers/teaReducers.js

```diff
@@ -1,4 +1,4 @@
-export default (state = { allTeas: {}, teaIDs: [] }, action) => {
+export default (state = { allTeas: {}, teaIDs: [] }, action = {}) => {
```

Comment:
There shouldn't be a case in which this reducer is called without an action - any reason why you added a default?

Reply:
I did add a test for a case where the reducer is called without an action at which point the default state should be returned. Without a default action, the tests fail, which seems to mean the action-less reducer would break as well.

Reply:
My point is, the reducer will never be called without an action unless there's a bug in your code, which you would then fix. I think it's OK for code to error out instead of handling being called incorrectly.

Reply:
Played with this a bit more and I understand it better now. Fixed.